libs/blocks/marketo-config/marketo-config.js

```diff
@@ -104,21 +113,71 @@ const validateState = (state, panelsData) => {
     });
   });
 
+  if ('form.fldStepPref' in state && state['form.fldStepPref']?.[2]?.length > 0) {
```

Comment:
`state['form.fldStepPref']?.[2]?.length` might be nice to rename to a variable for readability.